src/components/Contact/index.js

```diff
@@ -142,111 +183,48 @@ export default function ContactComponent() {
-                <label
-                  htmlFor="email"
-                  className="block text-sm font-semibold leading-6 text-gray-900"
-                >
-                  Email
-                </label>
-                <div className="mt-2.5">
-                  <input
-                    type="email"
-                    name="email"
-                    id="email"
-                    className="block w-full rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
-                    value={formData.email}
-                    onChange={handleChange}
-                    required={true}
-                  />
-                </div>
-              </div>
-              <div className="mt-2.5">
-                <label
-                  htmlFor="phoneNumber"
-                  className="block text-sm font-semibold leading-6 text-gray-900"
-                >
-                  Phone number
-                </label>
-                <div className="mt-2.5">
-                  <input
-                    type="tel"
-                    name="phoneNumber"
-                    id="phoneNumber"
-                    className="block w-full rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
-                    value={formData.phoneNumber}
-                    onChange={handleChange}
-                    required={true}
-                  />
-                </div>
-              </div>
-
-              <div className="sm:col-span-2">
-                <label
-                  htmlFor="message"
-                  className="block text-sm font-semibold leading-6 text-gray-900"
-                >
-                  Message
-                </label>
-                <div className="mt-2.5">
-                  <textarea
-                    name="message"
-                    id="message"
-                    rows={4}
-                    className="block w-full rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
-                    value={formData.message}
-                    onChange={handleChange}
-                    required={true}
-                  />
+              {formConfig.map((field) => (
+                <div className="mt-2.5" key={field.name}>
+                  <Label
+                    className="block text-sm font-semibold leading-6 text-gray-900"
+                    htmlFor={field.name}
+                  >
+                    {field.label}
+                  </Label>
+                  <div className="mt-2.5">
+                    {field.isTextarea ? (
+                      <textarea

```

Comment:
https://ui.shadcn.com/docs/components/textarea

we could also go for their textarea